tests/common/database_fixture.cpp

```diff
@@ -451,7 +451,7 @@ const asset_object& database_fixture::create_bitasset(
    creator.issuer = issuer;
    creator.fee = asset();
    creator.symbol = name;
-   creator.common_options.max_supply = GRAPHENE_MAX_SHARE_SUPPLY;
+   creator.common_options.max_supply = GRAPHENE_MAX_SHARE_SUPPLY / 2;
```

Comment:
Why? This may change the behaviour of existing tests.

Reply:
It does change behavior of existing tests, but without affecting their results (all existing tests pass).

Making this change helps prove that operations obey max_supply, and not just GRAPHENE_MAX_SHARE_SUPPLY. 

IMO, allowing a number here that is below GRAPHENE_MAX_SHARE_SUPPLY permits testing that is more narrow than the original setting.

Reply:
Probably ok. Can't think of a case where this would be bad.